homepage/homepage.css

```diff
@@ -0,0 +1,185 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    --product-name: #ff2020;
+    --primary-color: #e72744;
+    --secondary-color: #F05524;
+    --primary-btn: #6495ED;
+}
+
+
+/* nav bar */
+
+.landing-page-nav {
+    display: flex;
+    justify-content: space-between;
+    background-color: #f1f1f1;
+    padding: 10px;
+    margin-bottom: 20px;
+    margin-top: 0;
+}
+
+.nav-links {
+    padding-top: 20px;
+}
+
+.nav-links a {
+    color: black;
+    text-decoration: none;
+    padding-right: 15px;
+}
+
+.brand-name {
+    color: var(--primary-color);
+    font-size: 2.5rem;
+    font-weight: bold;
+}
+
+.brand-sub-title {
+    color: var(--secondary-color);
+    margin-top: -5px;
+    margin-left: 1rem;
+}
+
+.search-bar {
+    max-width: 800px;
+    width: 100%;
+}
+
+.form {
+    display: flex;
+}
+
+.search-bar-input {
+    flex-grow: 1;
+    padding: 16px;
+}
+
+.submit-btn {
+    padding: 15px;
+}
+
+.nav-icons {
+    display: flex;
+    justify-content: flex-end;
+    gap: 1.2rem;
+}
+
+.nav-icons a {
+    font-size: 24px;
+    text-decoration: none;
+    margin-top: 10px;
+    color: black;
+}
+
+
+/* image */
+
+.landing-page-image {
+    position: relative;
+}
+
+.main-img {
+    width: 100%;
+    height: 50vh;
+}
```

Comment:
On main img use object-fit : cover, it will set respective parent and will look better.

Reply:
> On main img use object-fit : cover, it will set respective parent and will look better.

thanks miheer will look on it